### PR TITLE
Update navigation part of UG

### PR DESCRIPTION
**Pull request by tsh22**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
...
 ---
 
+## Structure of the User Guide
+
+Worries about how tedious it will be to read this document? Not to worry! We have structured this User Guide in a way
+that makes it easy and quick for you to find what you need. In this next subsection,
+[Reading this User Guide](#reading-this-user-guide), you can find some tips we have on how to read this guide. The
+next section, [Features](#features), documents the main functionalities of _Teaching Assistant_ and how to use them.
+
+The main functions of _Teaching Assistant_ can be summmarised as the following:
```
Comment (tsh22) [inline docs/UserGuide.md:83]: Not sure if this part is needed, would it be repeating what's already in the intro?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
...
+
+![Annotated GUI](images/annotatedUi.png)
+
+The table below defines some technical terminology used throughout the User Guide.
+
+Terms | Meaning
+----------------- | -------------------
+GUI | Graphical User Interface. It refers to the part of the application you interact with.
+JSON | JavaScript Object Notation. Your Teaching Assistant data is stored as a JSON file format. You can find out more on their official website [here](https://www.json.org/json-en.html)!
+
```
Comment (goatygoatygoat) [inline docs/UserGuide.md:106]: Cuz I explained a bit of the terms GUI and CLI in the intro, maybe you want to shift the explanations to this table instead and remove the explanation in the intro

File: docs/UserGuide.md [hunk 3]
```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
...
+##### General Syntax
+The table below explains the general syntax used throughout the User Guide.
+
+Syntax | Meaning
+----------------- | -------------------
+`command` | This markup is used to specify text that can be entered into the command box.
+italics | Italics is used to represent terminology that is only specific to Teaching Assistant.
+<div markdown="span" class="alert alert-primary">:bulb:</div> | A bulb indicates that the following text is a tip.
+<div markdown="span" class="alert alert-warning">:exclamation:</div> | An exclamation mark indicates that the following text is a warning.
+
```
Comment (goatygoatygoat) [inline docs/UserGuide.md:116]: We also used the information source icon in features, maybe you can add this in

**Review by nicholastanvis — APPROVED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -68,6 +72,48 @@ An image of our UI is shown below!
...
+GUI | Graphical User Interface. It refers to the part of the application you interact with.
+JSON | JavaScript Object Notation. Your Teaching Assistant data is stored as a JSON file format. You can find out more on their official website [here](https://www.json.org/json-en.html)!
+
+##### General Syntax
+The table below explains the general syntax used throughout the User Guide.
+
+Syntax | Meaning
+----------------- | -------------------
+`command` | This markup is used to specify text that can be entered into the command box.
+italics | Italics is used to represent terminology that is only specific to Teaching Assistant.
```
Comment (goatygoatygoat) [inline docs/UserGuide.md]: tbh we never really use italics so this can go haha

File: docs/UserGuide.md (hunk 1 above)
Comment (goatygoatygoat) [reply docs/UserGuide.md:82]: can re-emphasise i guess

**Review by goatygoatygoat — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (goatygoatygoat) [reply docs/UserGuide.md:83]: can re-emphasise haha

Comment (tsh22) [reply docs/UserGuide.md:83]: okies

**Review by lrj689 — APPROVED** [review]

**Review by goatygoatygoat — APPROVED** [review]

**Pull request merged**